S/svg2pdf/build_tarballs.jl

```diff
@@ -0,0 +1,52 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "svg2pdf"
+version = v"0.10.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/typst/svg2pdf.git", "04424512b519bc0b141775b2aee1e82c68293b5b")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/svg2pdf
+cargo build -p svg2pdf-cli --release
+install -Dvm 755 "target/${rust_target}/release/svg2pdf${exeext}" "${bindir}/svg2pdf${exeext}"
+install_license ${WORKSPACE}/srcdir/svg2pdf/LICENSE-MIT
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("i686", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("powerpc64le", "linux"; libc = "glibc"),
+    Platform("i686", "linux"; libc = "musl"),
+    Platform("x86_64", "linux"; libc = "musl"),
+    Platform("aarch64", "linux"; libc = "musl"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("x86_64", "macos"; ),
+    Platform("aarch64", "macos"; ),
+    Platform("x86_64", "freebsd"; ),
+    Platform("x86_64", "windows"; )
+]
```

Comment:
```suggestion
platforms = supported_platforms()
```
This list appears to just be every platform we build on, correct? If not, there are enough here that it is better to do a filtering out instead of adding explicitly I think.

Reply:
Indeed it was only missing windows i686 as that failed on the wizard. I also fixed this using filter as suggested in the latest commit